packages/widget-chat/src/utils/files.js

```diff
@@ -16,3 +29,69 @@ export function bufferToBlob(buffer) {
   const objectUrl = urlCreator.createObjectURL(blob);
   return {blob, objectUrl};
 }
+
+export function constructFile(file) {
+  return _.assign(file, {
+    clientTempId: uuid.v4(),
+    displayName: file.name,
+    fileSize: file.size,
+    fileSizePretty: bytesToSize(file.size),
+    mimeType: file.type
+  });
+}
+
+export function constructImage(file) {
+  if (!isImage(file)) {
```

Comment:
Good catch. Turns out that method isn't being used anywhere anymore... so just gonna get rid of it completely.